src/test/resources/edu/hm/hafner/analysis/parser/issue73509.txt

```diff
@@ -0,0 +1,9 @@
+In member function 'void CInput::UpdateText(int, int, int)':
+cc1plus: warning: 'void* _builtin_memset(void*, int, long unsigned int)': specified size 18446744073709551612
+exceeds maximum object size 9223372036854775807 [-Wstringop-overflow=]
+
+In member function 'void CInput::UpdateText(int, int, int)':
+cc1: warning: 'void* _builtin_memcpy(void*, const void*, long unsigned int)': specified size 18446744073709551612
+exceeds maximum object size 9223372036854775807 [-Wstringop-overflow=]
+
+testhist.l:451: warning: `void yyunput(int, char*)' defined but not used
```

Comment:
Is this output generated by an actual compiler call? Or did you just copy the text from the issue?